meta-lmp-base/recipes-core/dbus/dbus_%.bbappend

```diff
@@ -1,2 +1,16 @@
 # Avoid warnings with systemd
 EXTRA_OECONF += "--runstatedir=/run"
+
+FILESEXTRAPATHS:prepend := "${THISDIR}/${PN}:"
+
+SRC_URI += "file://tmpfiles.conf"
+
+do_install:append () {
+    if ${@bb.utils.contains('DISTRO_FEATURES', 'systemd', 'true', 'false', d)}; then
+        install -d ${D}${nonarch_libdir}/tmpfiles.d
+        install -m 0644 ${WORKDIR}/tmpfiles.conf ${D}${nonarch_libdir}/tmpfiles.d/${PN}.conf
+        rm -rf ${D}${localstatedir}
+    fi
+}
+
+FILES:${PN}:remove = "${@bb.utils.contains('DISTRO_FEATURES', 'systemd', '${localstatedir}', '', d)}"
```

Comment:
Should be fine to let localstatedir on FILES as the build won't complain about missing files/directories.

It just complains when it installs additional files/dirs and they are not reflected on a particular FILES var entry.